packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -43,10 +57,16 @@ export default function CreateAccount (): React.ReactElement {
 
   const _onNextStep = useCallback(() => setStep((step) => step + 1), []);
   const _onPreviousStep = useCallback(() => setStep((step) => step - 1), []);
+  const _onCancel = useCallback(() => {
+    flushAccountCache().catch((e) => console.error(e));
+    onAction('/', { resetCachedAccount: true });
```

Comment:
If you are passing the reset flag, why are we explicitly flushing the cache here and above? While the flag feels a bit messy (it is growing on me), just making it explicit and using it everywhere is the better/cleaner/more-maintainable option